pkg/standalone-utils/test/VaultActions.test.ts

```diff
@@ -97,12 +98,17 @@ describe('VaultActions', function () {
     outputReferences?: Dictionary<BigNumberish>;
     sender: Account;
     recipient?: Account;
+    useInternalBalance?: boolean;
```

Comment:
There was previously no way to exit to internal balance (in the tests). It was hard-coded to false. Since I needed to do this in one of the tests, I had to add it. It defaults to the previously hard-coded value, so that the existing tests don't change.